### ISSUE TITLE
Keyboard focus skips pdf content

### ISSUE THREAD

**Issue by Zoidster** [stephanrauh/ngx-extended-pdf-viewer#745]
**Describe the bug**
When tabbing through the controls of the viewer, the focus jumps from the *tools* button directly to the next focus element after the viewer. The content never receives the focus, so scrolling the pdf with the keyboard is impossible.
This is an issue for accessibility. Is there any documentation for the accessibility of this component?

**Version info**
- ngx-extended-pdf-viewer 8.3.0
- Angular 11.2.9 

**Desktop:**
 - Browser: Chrome

**To Reproduce**
Reproduction project: [ngx-pdf-issue.zip](https://github.com/stephanrauh/ngx-extended-pdf-viewer/files/6425983/ngx-pdf-issue.zip)
I didn't change much, besides running npm update and updating to Angular 11.2.9
The behaviour is independent of the pdf file displayed.

**Screenshots**
As shown by the [chrome extension taba11y](https://chrome.google.com/webstore/detail/taba11y/aocppmckdocdjkphmofnklcjhdidgmga?hl=en) there is no tabindex set for the pdf content:
![pdf_viewer_taborder](https://user-images.githubusercontent.com/924818/117116683-f0180780-ad8e-11eb-83d3-bdb51a13cfbe.PNG)

**Comment by stephanrauh** [comment]
Thanks for pointing this out! I have to admit I neglected this topic for a while. Some time ago, I made sure that the sidebar can be used with the keyboard, but even that may be broken now.

As for the documentation: at the moment I don't know how to document accessibility. What kind of documentation do you have in mind?

**Comment by Zoidster** [comment]
I'm really not an expert in the area but I enjoy unsing the [angular material a11y documentation sections](https://material.angular.io/components/expansion/overview#accessibility).
They document what the components do on their own to ensure basic accessibility and what the user needs to do for correct usage (for example add aria labels to a component)

For my project I need to be compliant with WCAG 2.1 level AAA, so any information regarding compliance to that standard would be helpful.
Thank you so much for not dismissing accessibility!

**Comment by stephanrauh** [comment]
The enhancement is going to land this evening with version 10.0.0-alpha.7.
